synapse/handlers/cas.py

```diff
@@ -120,7 +120,7 @@ async def _validate_ticket(
         Returns:
             The parsed CAS response.
         """
-        uri = self._cas_server_url + "/proxyValidate"
+        uri = self._cas_server_url + "/p3/proxyValidate"
```

Comment:
Will this break non-v3 servers? Do we need to fallback to this URL or have configuration?

Reply:
I've no way to test with v2.
I'm adding a configuration for v3, that will be better ...